README.md

```diff
@@ -1,2 +1,43 @@
 # sky9
 AWS accounts and users management
+
```

Comment:
Please mention the prerequisites for all the things below. Like: NodeJS, Serverless, configured AWS account.

Reply:
Done.